misc/klayout/force-python-dynamic-link.patch

```diff
@@ -0,0 +1,53 @@
+diff --git a/build.sh b/build.sh
```

Comment:
Log a github issue to get this upstream

Reply:
I suspect @klayoutmatthias link statically intentionally.